browser/src/control/Control.ContextMenu.js

```diff
@@ -309,9 +309,13 @@ L.Control.ContextMenu = L.Control.extend({
 					itemName = _UNO(item.command, docType, true);
 				}
 
+				var toSnakeCase = function (text) {
+					return text.replace(/[ _]/gi, '-').replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase().replace('...', '');
```

Comment:
what if we have more than 2 words? or single one?

can't we just do `split` and then `join` ?

Reply:
'/g' ensures it applies on all the string:

toSnakeCase("InserAnnotationTest")
=> "inser-annotation-test"

toSnakeCase("Insert test")
=> "insert-test"

toSnakeCase("Insert")
=> "insert"